eq_translations/utils.py

```diff
@@ -140,5 +111,40 @@ def remove_quotes(message):
     return message.strip()
 
 
-def are_dumb_strings_equal(message_a, message_b):
+def are_dumb_strings_equal(message_a, message_b, pluralizable=False):
```

Comment:
I think the plural tuples should be handled in another method.

Reply:
This is now a thing of the past.